pipeline_dp/pipeline_operations.py

```diff
@@ -157,6 +161,19 @@ def count_per_element(self, rdd, stage_name: str = None):
         return rdd.map(lambda x: (x, 1))\
             .reduceByKey(lambda x, y: (x + y))
 
+    def reduce_accumulators_per_key(self, rdd, stage_name: str = None):
+        """Reduce the input RDD so that all elements per each key are merged.
```

Comment:
Please move this comment to a base class (just replacing, RDD -> collections). The idea that the code that uses PipelineOperations don't need to know which framework they use

Reply:
Good idea. Updated.